lib/impure/db_sqlite.nim

```diff
@@ -642,7 +635,7 @@ proc getValue*(db: DbConn,  stmtName: SqlPrepared): string
 
 proc tryInsertID*(db: DbConn, query: SqlQuery,
                   args: varargs[string, `$`]): int64
-                  {.tags: [WriteDbEffect], raises: [].} =
```

Comment:
Instead of removing this `raises`, maybe better to explicitly write out a list of exceptions that can be raised?

Reply:
Yeah, I'm not sure about this. Maybe we should turn raise errors into warnings first to reduce breaking changes. 

I will add it anyway.

Reply:
solved